examples/Example21/src/AdePTTrackingManager.cc

```diff
@@ -0,0 +1,84 @@
+#include "AdePTTrackingManager.hh"
+
+#include "G4Threading.hh"
+#include "G4Track.hh"
+#include "G4EventManager.hh"
+#include "G4Event.hh"
+#include "G4RunManager.hh"
+
+#include "G4Electron.hh"
+#include "G4Gamma.hh"
+#include "G4Positron.hh"
+
+
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......
+
+AdePTTrackingManager::AdePTTrackingManager() {
+}
+
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......
+
+AdePTTrackingManager::~AdePTTrackingManager() {
+   fAdept->Cleanup();
+}
+
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......
+
+void AdePTTrackingManager::BuildPhysicsTable(const G4ParticleDefinition &part) {
```

Comment:
This function is called once per particle type (e-/+/gamma); it needs a protection to only initialize once.